package.json

```diff
@@ -24,27 +24,27 @@
     "async": "^0.9.0",
     "base64id": "0.1.0",
     "basic-auth-connect": "^1.0.0",
-    "body-parser": "^1.12.0",
+    "body-parser": "^1.19.0",
     "connect-redis": "^2.1.0",
-    "cookie-parser": "^1.3.3",
-    "express": "^4.10.1",
-    "express-session": "^1.9.1",
+    "cookie-parser": "^1.4.5",
+    "express": "^4.17.1",
+    "express-session": "^1.17.1",
     "logger-sharelatex": "^1.7.0",
     "metrics-sharelatex": "^2.6.2",
-    "redis-sharelatex": "^1.0.12",
-    "request": "^2.88.0",
+    "redis-sharelatex": "^1.0.13",
```

Comment:
Can we please hold-off on the `redis-sharelatex` upgrade to separate any issues from the `ioredis`upgrade? It is scheduled as part of https://github.com/overleaf/issues/issues/3095 cc @briangough

Reply:
sure!